tests/testthat/test_variable_label.R

```diff
@@ -140,19 +52,96 @@ test_that(
 context("variable_label() extraction methods")
 
 test_that(
-  "variable_label,labelled-method"
+  "variable_label.labelled-method"
   , {
     object <- 1:10
     class(object) <- c("labelled", "integer")
     attr(object, "label") <- "label1"
-    variable_label(object)
+
     expect_identical(
       object = variable_label(object)
       , expected = "label1"
     )
   }
 )
 
+context("methods for class .labelled")
+
+test_that(
+  "droplevels.labelled-method"
+  , {
+    # only check consitency, as base behavior has changed recently
+    x <- factor(letters[1:4], levels = letters[1:10])
+    variable_label(x) <- "Test me!"
+    x <- droplevels(x, exclude = "d")
+
+    y <- factor(letters[1:4], levels = letters[1:10])
+    y <- droplevels(y, exclude = "d")
+    variable_label(y) <- "Test me!"
+
+    expect_identical(
+      object = x
+      , expected = y
+    )
+  }
+)
+
+test_that(
```

Comment:
For some reason, this test fails for me when I run it manually. The indexing method also does not work in the console for me. Any idea what might me happening?

Reply:
Did you mean the "`[`.labelled-method" test? This one should work now

Reply:
Yes, that's what I meant.